### PR TITLE
minor improvements

### PR DESCRIPTION
**Pull request by eigenpaul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
 
 ## Limitations (to be implemented)
 
 - ClusterClass and Cluster Stack templates are [hard-coded](src/components/cluster-form.tsx#L90) and limited to the `scs/alpha` Cluster Stacks
   - Ideally they should reflect the current state of each specific Cluster Stack (e.g. metal3/kamaji)
-  - Possible solution: Use GitHub API to read and then parse the ClusterClasses. Maybe build upon on [this](https://github.com/SovereignCloudStack/cluster-stacks/pull/94)
+  - Possbile solution: Either fetch and parse the ClusterClasses using the GitHub API or use the schema provided by [kube-apiserver](https://github.com/SovereignCloudStack/cluster-gen/issues/3)
 - Cluster Stack releases are [hard-coded](src/components/clusterstack-form.tsx#L116)
   - Possible solution: Use GitHub API to get releases
-- Instance flavors are [hard-coded](src/components/cluster-form.tsx#L347)
```
Comment (DEiselt) [inline README.md:18]: Why is this removed? As of right now, they are still hardcoded and it would be desirable to load them dynamically (independent of how often they change).

File: src/components/cluster-form.tsx [hunk 2]
```diff
@@ -408,7 +408,7 @@ export function ClusterForm() {
               name="worker_flavor"
               render={({ field }) => (
                 <FormItem>
-                  <FormLabel className="text-m">Worker Flavor</FormLabel>
+                  <FormLabel className="text-m">Worker Flavor <span className='border-2 rounded-3xl bg-gray-200 text-black px-2' title="SCS-( CPU )-( RAM[GiB] )-( DISK[GB] )">i</span></FormLabel>
```
Comment (DEiselt) [inline src/components/cluster-form.tsx:411]: Good catch, we need to make a component :D

File: README.md (hunk 1 above)
Comment (eigenpaul) [reply README.md:18]: True, I just figured it's not that of a pressing issue anymore, since we now have all available flavors of the standard

File: src/components/cluster-form.tsx (hunk 2 above)
Comment (eigenpaul) [reply src/components/cluster-form.tsx:411]: Maybe [this one](https://ui.shadcn.com/docs/components/tooltip)

**Review by DEiselt — APPROVED** [review]
LGTM
